extension/src/setup/index.ts

```diff
@@ -499,6 +501,16 @@ export class Setup
     }
   }
 
+  private async updatePythonEnvironment() {
+    const value = await pickPythonExtensionAction()
+
+    if (!value) {
+      return
+    }
+
+    return value === 1 ? createPythonEnv() : selectPythonInterpreter()
```

Comment:
[I] Let's make the values members of an enum so that they are connected. (i.e if 1 changes in `pickPythonExtensionAction` it will automatically be updated here. It would be good to name them appropriately as well.

Reply:
Good idea!